lib/utility/analytics.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:firebase_analytics/firebase_analytics.dart';
+
+FirebaseAnalytics analytics = new FirebaseAnalytics();
```

Comment:
Stop doing this.

Please avoid using a global var in all the app.

Import properly Analytics Service with GetIt and use FirebaseAnalytics inside the Analytics Service

Using a service will give us the possibility to mock it in tests